File: src/main/java/ru/romanow/jpa/model/PersonResponse.java
package ru.romanow.jpa.model;

import lombok.Data;
import lombok.experimental.Accessors;

import javax.persistence.Column;

@Data
@Accessors(chain = true)
public class PersonResponse {
    private String firstName;
    private String middleName;
    private String lastName;
    private Integer age;
    private AddressInfo address;
}

File: src/main/java/ru/romanow/jpa/model/PersonResponse.kt
package ru.romanow.jpa.model

data class PersonResponse(
    override var id: Int? = null,
    var firstName: String? = null,
    var middleName: String? = null,
    var lastName: String? = null,
    var age: Int? = null,
    var address: AddressInfo? = null,
    var roles: Set<String>? = null,
    var authorities: Set<AuthorityInfo>? = null
) : IdentifiableModel
